pravega-client-examples/src/test/java/io/pravega/example/streamprocessing/WorkerProcessGroup.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package io.pravega.example.streamprocessing;
+
+import lombok.Builder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.IntStream;
+
+/**
+ * This manages a group of WorkerProcess instances.
+ */
+@Builder
+public class WorkerProcessGroup implements AutoCloseable {
```

Comment:
The value of this class is not clear to a casual reader.

Reply:
I added a comment.
```
 * This manages a group of WorkerProcess instances.
 * The testing framework in {@link StreamProcessingTest} uses this class to
 * start and stop multiple instances of {@link AtLeastOnceProcessorInstrumented}
 * in a single process.
```